source/actors/player/Player.hx

```diff
@@ -70,7 +78,9 @@ class Player extends FlxSprite {
 
 		touching = FlxObject.DOWN;
 		input = new Input();
-        
+
+        Enemy.addTarget(this);
```

Comment:
Rather than needing the player do this static setup for the enemy class, I think it's cleaner if enemies take a `player` object as a parameter, then do the static setup themselves in their constructor if necessary.